.github/workflows/sub_test_choice.yaml

```diff
@@ -59,6 +59,12 @@ on:
       os_to_test:
         required: true
         type: string
+      os_version_install:
+        description: OS version to install
```

Comment:
`description` is only defined in `master-e2e` file (and of course in the `workflow_dispatch` portions too). I did this to avoid the need of adding/modifying/fixing it multiple time, as previously we usually forgot to modify it when needed ;-)

But `required: true` should be added.